samples/snippets/pom.xml

```diff
@@ -1,47 +1,37 @@
-<?xml version='1.0' encoding='UTF-8'?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file is generated too, so you may need to opt it out - @chingor13 for guidance here

Reply:
the original code restored and all other project settings are added